Source/Core/VideoCommon/TextureCacheBase.cpp

```diff
@@ -459,6 +459,12 @@ void TextureCacheBase::SerializeTexture(AbstractTexture* tex, const TextureConfi
     // needing to allocate/free an extra buffer.
     u8* texture_data = p.DoExternal(total_size);
 
+    if (p.GetMode() == PointerWrap::MODE_MEASURE)
+    {
+      // Something happened, maybe not enough memory left in the PointerWrapped buffer.
```

Comment:
Should we have a log or something to call out this scenario?

Reply:
Sure, can't hurt.